benchmarks/meshoptimizer/build-native.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Build all of the shootout benchmarks as native shared libraries (Linux-only).
+#
+# Usage: ./build-native.sh
+
+(set -x; rm -r -f meshoptimizer);
+(set -x; git clone --recurse-submodules https://github.com/zeux/meshoptimizer.git);
```

Comment:
Will this newly-created directory be ignored by Git or should we add to a `.gitignore` in this directory?

Reply:
Yes .. it shows. Will add.